com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/LangServerBootstrapTest.java

```diff
@@ -78,6 +79,7 @@ public void isPipeEnabledInvalidArgument() {
   }
 
   @Test
+  @Ignore
```

Comment:
I will suggest to don't ignore it until we can still use socket communication (for developer purposes)

Reply:
Oops...